conseil-common/src/main/scala/tech/cryptonomic/conseil/common/mantis/Utils.scala

```diff
@@ -0,0 +1,30 @@
+package tech.cryptonomic.conseil.common.mantis
+
+import scala.util.Try
+
+import scorex.crypto.hash.Keccak256
+
+/**
+  * Utils methods required by the Mantis integration.
+  */
+object Utils {
+
+  /**
+    * Decode hex string.
+    */
+  def hexToString(value: String): String =
+    value.stripPrefix("0x").grouped(2).toArray.map(Integer.parseInt(_, 16).toChar).mkString.trim
+
+  /**
+    * Create SHA-3 signature from the first 4 bytes of the given string.
+    */
+  def keccak(value: String): String =
```

Comment:
why not in `conseil-common/src/main/scala/tech/cryptonomic/conseil/common/util/CryptoUtil.scala` ?

Reply:
i know it would require to update ethereum integration too. But i think is safe change.